lib/defines.include.php.sample

```diff
@@ -32,6 +32,7 @@ define('CSR_DOMAIN', 'csrdelft.nl');
 
 # urls ZONDER trailing slash
 define('CSR_ROOT', 'https://' . CSR_DOMAIN);
+define('ASSETS_DIR', CSR_ROOT . '/assets');
```

Comment:
Er wordt een `CsrLayoutPage` of `CsrOweeLayoutPage` aangemaakt, deze verwijst intern naar de goede assets. Binnen `CsrLayoutPage` wordt onderscheid gemaakt tussen verschillende thema's, maar deze worden op basis van naam van `CompressedResource` gekozen.